supervisor/resolution/checks/addon_pwned.py

```diff
@@ -26,6 +34,16 @@ def enabled(self) -> bool:
             return False
         return super().enabled
 
+    @enabled.setter
+    def enabled(self, value: bool) -> None:
+        """Enable or disbable check."""
+        if not self.sys_security.pwned and value is True:
+            raise ResolutionCheckError(
+                f"Can't enable '{self.slug}',' 'security.pwned' is disabled",
+                _LOGGER.warning,
+            )
+        self.sys_resolution.check.data[self.slug][ATTR_ENABLED] = value
```

Comment:
Look like that is a complete missunderstand. Let's ditch the enable overwrite complete and we don't allow anymore to modify that outside of the base. After that, we add on the run_check as first to check the global setting with a warning that the check gets skip because of the global setting.